row_081_u1.py

```
@@ -0,0 +1,499 @@
+# app1 = Flask(__name__)
+app = Flask(__name__)
+wsgi_app = app.wsgi_app
+
+
+
+mail=Mail(app)
+
+app.config['MAIL_SERVER']='smtp.gmail.com'
+app.config['MAIL_PORT'] = 465
+app.config['MAIL_USERNAME'] = 'bhatiaaryan14@gmail.com'
+app.config['MAIL_PASSWORD'] = 'plmqaz0657'
```

Ruff (0.13.2)

Possible hardcoded password assigned to: "MAIL_PASSWORD"

(S105)